vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp

```diff
@@ -56,6 +58,8 @@ $( document ).ready(function() {
 	$("#cancelSpaceLinkBtn").click(function() {
 		storeX = null;
 		storeY = null;
+		$("#arrow").remove();
+		$("#arrow").removeAttr("id");
```

Comment:
if #arrow is already removed, why do you remove the attribute as well?

Reply:
Will modify that.